tests/validation_tests/test_failures.py

```diff
@@ -457,3 +457,6 @@ def test_construct_cell_index():
 
     test_index11 = construct_cell_index("Outcome_Data", "Higher Frequency", [8, 2, 11, 5, 9])
     assert test_index11 == "P8, P2, P11, P5, P9"
+
```

Comment:
It feels like this test should be split into a few smaller ones. The inline comments allude to at least 3 different index types (single, multi etc) maybe those groupings would be a sensible split?

Reply:
Done